pr-data.csv

```diff
@@ -1170,6 +1216,14 @@ https://github.com/apache/commons-math,43b77ffabba8a48144080674973362150cbcfb28,
 https://github.com/apache/commons-math,43b77ffabba8a48144080674973362150cbcfb28,.,org.apache.commons.math4.ml.neuralnet.sofm.KohonenUpdateActionTest.testUpdate,ID,DeveloperFixed,,https://github.com/apache/commons-math/commit/3e72f1ea31f4b0ed43e6cb48381f86b95c15f36c
 https://github.com/apache/commons-math,ec7eb8b32d9dff80f98c506a88c521a0ca3ea6a6,commons-math-neuralnet,org.apache.commons.math4.neuralnet.NetworkTest.testIterationOrder,ID,InspiredAFix,https://github.com/apache/commons-math/pull/213,https://github.com/apache/commons-math/commit/50977290cdda79896ccb6304067341486fe104d2
 https://github.com/apache/commons-math,ec7eb8b32d9dff80f98c506a88c521a0ca3ea6a6,commons-math-neuralnet,org.apache.commons.math4.neuralnet.sofm.KohonenUpdateActionTest.testUpdate,ID,InspiredAFix,https://github.com/apache/commons-math/pull/213,https://github.com/apache/commons-math/commit/3e72f1ea31f4b0ed43e6cb48381f86b95c15f36c
+https://github.com/apache/curator,f5fb6d109257e35de250af7d6c0090b42f2a18a2,framework,org.apache.curator.framework.imps.TestReconfiguration.testAdd,ID,Accepted,https://github.com/apache/curator/pull/491,
+https://github.com/apache/curator,f5fb6d109257e35de250af7d6c0090b42f2a18a2,framework,org.apache.curator.framework.imps.TestReconfiguration.testAddAndRemove,ID,Accepted,https://github.com/apache/curator/pull/491,
+https://github.com/apache/curator,f5fb6d109257e35de250af7d6c0090b42f2a18a2,framework,org.apache.curator.framework.imps.TestReconfiguration.testAddAndRemoveWithEmptyList,ID,Accepted,https://github.com/apache/curator/pull/491,
+https://github.com/apache/curator,f5fb6d109257e35de250af7d6c0090b42f2a18a2,framework,org.apache.curator.framework.imps.TestReconfiguration.testAddAsync,ID,Accepted,https://github.com/apache/curator/pull/491,
+https://github.com/apache/curator,f5fb6d109257e35de250af7d6c0090b42f2a18a2,framework,org.apache.curator.framework.imps.TestReconfiguration.testBasicGetConfig,ID,Accepted,https://github.com/apache/curator/pull/491,
+https://github.com/apache/curator,f5fb6d109257e35de250af7d6c0090b42f2a18a2,framework,org.apache.curator.framework.imps.TestReconfiguration.testNewMembersWithEmptyList,ID,Accepted,https://github.com/apache/curator/pull/491,
+https://github.com/apache/curator,f5fb6d109257e35de250af7d6c0090b42f2a18a2,framework,org.apache.curator.framework.imps.TestReconfiguration.testRemoveWithChroot,ID,Accepted,https://github.com/apache/curator/pull/491,
+https://github.com/apache/curator,f5fb6d109257e35de250af7d6c0090b42f2a18a2,framework,org.apache.curator.x.discovery.details.TestJsonInstanceSerializerCompatibility.testCompatibilityMode,ID,Accepted,https://github.com/apache/curator/pull/491,
```

Comment:
Although the developer said lgtm but the PR has not been merged yet, can you ping the developer again?

Reply:
For both these two the changes are all "accepted"

<img width="762" alt="image" src="https://github.com/TestingResearchIllinois/idoft/assets/54618402/ecf37788-7eac-4452-88a4-ae9f08c76ef1">

Reply:
They both approved these two --- which means they are both accepted!!!